CONTRIBUTING.md

```
@@ -60,6 +187,150 @@ You must use your real name (sorry, no pseudonyms or anonymous contributions).
+For just the most recent commit:
+
+```bash
+git commit --amend -s --no-edit
+git push --force-with-lease
+```
+
+## Commit messages
+
+Use [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)